### PR TITLE
ENH: add interactive mode

### LINKED ISSUE #38: Interactive mode
Have an import for running interactively in ipython

### PR DESCRIPTION
**Pull request by CJ-Wright**
Closes #38

### CODE REVIEW

**Comment by scopatz** [comment]
Looks reasonable to me, but could use a top-level docstring and some example usage patterns

File: regolith/interact.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""
+Loads the dbs for interactive sessions
+
+Usage
```
Comment (CJ-Wright) [inline regolith/interact.py]: Will this render correctly with automodule?

Comment (scopatz) [reply regolith/interact.py]: Probably not, you want the following to be a code-block, right?

**Comment by CJ-Wright** [comment]
@scopatz ready for review!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**